lib/league_stats.rb

```diff
@@ -0,0 +1,134 @@
+require 'csv'
+require 'team'
+require 'game_teams'
+require 'game'
+
+class LeagueStats
+  attr_reader :teams,
+              :game_teams,
+              :games
```

Comment:
I don't think these are needed either since all the calls to these attributes are being done within this class.

Reply:
I  agree with you this is a habit and not necessarily a thoughtful piece of code